src/main.rs

```diff
@@ -2,9 +2,8 @@
 
 use clap::Parser;
 use cli::SubCommands;
-use std::time::Duration;
-
 use rumqttc::{self, Client, MqttOptions, QoS, Transport};
+use std::time::Duration;
```

Comment:
I moved the `std` import up in bc60f94f250574db6e5c3f4aa34dde2540ad944b which creates a merge confict. Tried to fix it for you but don't seem to have push access to the PR. Just as a heads up that that's the reason for the merge conflict so you have an easier time to fix it :)

Reply:
Likely that's because I was working on `main`

Reply:
Addressed in https://github.com/EdJoPaTo/mqttui/pull/102/commits/3042f54b11aa4c06388797a8191d41d7f9733e75